internal/validation/mock_trace_validation.go

```diff
@@ -92,6 +123,117 @@ func AddSpans(uploadedSpans map[string]*cloudtrace.Span, lock *sync.Mutex,
 	return nil
 }
 
+// ValidateProjectName verifies that the project name from the BatchWriteSpans request
+// is of the form projects/[PROJECT_ID]
+func ValidateProjectName(projectName string) error {
+	if !projectNameRegex.MatchString(projectName) {
+		return statusInvalidProjectName
+	}
+	return nil
+}
+
+// A display name can be at most 128 bytes.
```

Comment:
nit: method comments should start with the name of the method

Reply:
Done